public/components/all/package.go

```diff
@@ -7,4 +7,5 @@ import (
 	_ "github.com/wombatwisdom/wombat/public/components/redpanda"
 	_ "github.com/wombatwisdom/wombat/public/components/snowflake"
 	_ "github.com/wombatwisdom/wombat/public/components/splunk"
+	_ "github.com/wombatwisdom/wombat/public/components/zeromq"
```

Comment:
What do you think on moving zeromq to an extension package instead of all? Reasoning behind it is that as far as I know, zeromq requires cgo while wombat currently does not require it.